buildscripts/condarecipe.local/meta.yaml

```diff
@@ -63,6 +63,8 @@ requirements:
     - scipy >=1.0
     # CUDA Python 11.6 or later
     - cuda-python >=11.6
+    # numba-rvsdg
+    - numba-rvsdg 0.0.*      # [py==311]
```

Comment:
I'd suggest we don't pin to any specific version so that we have rooms to fix stuff without needing to re-release.  Also, this is just a test run for the dependency. Only bleeding-edge developers will be using `numba-rvsdg` and they can manually pin versions if needed.

Reply:
we should pin to `0.0.5` once it's released via https://github.com/numba/numba-rvsdg/pull/105